app/services/metrics/base.rb

```diff
@@ -1,28 +1,26 @@
 module Metrics
   class Base < BaseService
-    def calculate_metrics_avg(entities, metric_type)
-      entities.reject { |_entity, count| count == 1 }.each do |entity, count|
-        Metric.find_by!(ownable: entity, value_timestamp: metric_interval, name: metric_type)
-              .tap do |metric|
-          metric.value = metric.value / count
-          metric.save!
-        end
-      end
-    end
+    Metric = Struct.new(:ownable_id, :ownable_type, :value_timestamp, :name, :value)
```

Comment:
What is this for?

Reply:
I use structs to return results in a defined way. In the last commit, I delete some of the attributes.